packages/chat/src/ChatClient.ts

```diff
@@ -213,8 +216,9 @@ export class ChatClient extends IrcClient {
 	 *
 	 * @eventListener
 	 * @param channel The channel whose chat is cleared.
+	 * @param channelId The channel ID whose chat is cleared.
 	 */
-	readonly onChatClear: EventBinder<[channel: string]> = this.registerEvent();
+	readonly onChatClear: EventBinder<[channel: string, channelId: string]> = this.registerEvent();
```

Comment:
I don't see why you would pass the channel ID here, where on the other two events you added the message object.

Reply:
Because `onChatClear` only emits the channel [as seen here](https://github.com/twurple/twurple/blob/975f90af3e49dc80336c178ad2796a38478ed63d/packages/chat/src/ChatClient.ts#L847) and it does not return `targetUserId`